docs/semgrep-appsec-platform/dashboard-beta.md

```diff
@@ -0,0 +1,121 @@
+
+- Evaluation of the success of your AppSec program, enabling you to know your current security risk.
+  - This includes assessing the deployment of secure guardrails to your organization.
+- Awareness of trends and opportunities that you can use to improve your security posture.
+  - You are able to view **recurring security issues**, consequently taking action on them.
+
+<!-- 
+The dashboard provides the following features:
+
+- Filters that enable you to track key metrics over a period of time, per Semgrep product, and per project.
+-->
+
+:::note
+This feature is in a **private beta**. To request access, contact your Technical Account Manager or your Account Executive and let them know you'd like to join the dashboard beta.
+:::
+
+
+![Dashboard (beta) page](/img/dashboard-fold.png)
+_**Figure**. The dashboard (beta_) page._
+
+The dashboard is divided into several panes:
+
+<dl>
+<dt>Filters and configuration</dt>
+<dd>Sets the time period, product, and project scope of the <strong>production backlog</strong> and <strong>secure guardrails</strong> charts in the page.</dd>
+<dt>Production backlog</dt>
+<dd>Displays data about all the findings detected in your primary or default branch.</dd>
+<dt>Secure guardrails</dt>
+<dd>Displays data relevant to the deployment of secure guardrails.</dd>
+<dt>Most findings by project</dt>
+<dd>Lists findings by project, grouped by **product** or **severity**.</dd>
+<dt>Mean time to remediate</dt><!-- Median open finding age -->
+<dd>A graph showing the time to remediate over the given time period.</dd>
+</dl>
+
+## Filters and configuration
+
+Use the filters to gain a top-level view or zoom in to a specific period of time. Create quarterly overviews or recent incident statements for various AppSec stakeholders by specifying the following filters:
+
+- Time period
+- Project (repository)
+- Semgrep product or type of scan (SAST, SCA, or Secrets)
+
+Configurations set here apply to the following panes:
+
+- Production backlog
+- Secure guardrails
+
+### Recommended priority filters
+
+By default, **<i class="fa-solid fa-toggle-large-on"></i> Recommended priority** filters are enabled. These filters display any finding that is **Critical** or **High** severity in **addition** to being:
+
+- High confidence - if the finding is from Semgrep Code.
+- Reachable - if the finding is from Semgrep Supply Chain.
+- Valid - if the finding is from Secrets.
+
+tk add links to definitions
+
+## Production backlog
+
+This pane displays analytics related to findings detected in your primary or default branch. This typically means that the finding, usually a security issue, has made it to production environments.
+
+### Key metrics 
+
+| Key metrics    | Description |
+| -------------- | ------ |
+| Total new      | |
+| Total fixed    | |
+| Total ignored  | |
+| Total net new  | |
```

Comment:
How is **Total net new** computed as opposed to **Total new**?

Reply:
total net new = total new - (total amount of fixed + total amount of ignored). we can probably add some context about the goal being to get total net new as close to zero as possible, meaning that you're addressing things as quickly as new things are coming in

Reply:
Thank you! 
![image](https://github.com/user-attachments/assets/00a58cc3-58e3-4d45-9432-7b0d5cdbdb84)